src/aead.rs

```diff
@@ -208,9 +211,10 @@ impl PartialEq for Algorithm {
 
 impl Eq for Algorithm {}
 
-/// An authentication tag.
+/// A possibly valid authentication tag.
 #[must_use]
 #[repr(C)]
+#[derive(Clone, Copy)]
 pub struct Tag([u8; TAG_LEN]);
```

Comment:
Use test `aead_test_aad_traits` as a model for covering the `Clone` and `Copy` implementations of `Tag`.

Reply:
Thanks for pointing those out, I've added the test coverage, along with some more cases for the `open_in_place_separate_tag` function itself.